packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/l10n/network_lookup/field_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nFieldLookup extends I18nFieldLookup {
           bundledI18n = I18nField(I18nFieldLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nField(I18nFieldLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong